experiments/SlabLand/README.md

```diff
@@ -0,0 +1,155 @@
+
+        function numerical_boundary_flux_second_order!(
+            numerical_flux::Union{PenaltyNumFluxDiffusive},
+            bctype::CoupledPrimaryBoundary,
+            balance_law::ModelSetup,
+            fluxᵀn::Vars, _...)
+            F_tot = calculate_land_sfc_fluxes(balance_law, state_prognostic⁻, state_auxiliary⁻, t)  # W/m^2
+            fluxᵀn.ρθ = - F_tot  / balance_law.parameters.cp_d
+
+        end
+
+## **Slab Land Component**
+
+For prototyping of the coupler, we parameterize the land as a 0-dimensional slab covering the whole globe, analogous to the slab ocean with no water. 
+
+### 1. Formulation
+The evolution of the prognostic variable, surface soil temperature ($T_{sfc}$), can be written in the units of heat flux (W m$^{-2}$):
+
+$$
+\rho_s h_s c_s \frac{\partial T_{sfc}}{\partial t} = F_{tot} + G = R_{SW} - R_{LW} - SH - LH + G
+$$
+
+  
+where
+
+- Sensible heat flux at surface
+    $$SH = cp_d g_a (T_{sfc} - T_a) = cp_d g_a (T_{sfc} - T_a)$$
+- Latent heat flux at surface
+    $$LH = \lambda g_w (q_{sat}(T_{sfc},p_{sfc}) - q_a)$$
+- Downward-pointing short-wave radiative flux
+    $$R_{SW} = F_{sfc}^d - F_{sfc}^u = (1-\alpha)\tau F_{sol} * (1 + \sin(2π τ_d^{-1} t))$$
+- Upward-pointing net long-wave flux
+    $$R_{LW} = F_{sfc}^u - F_{sfc}^d = \epsilon \sigma \theta_{sfc}^4 - \epsilon F_{a}$$
+- Soil storage
+    $$G = \kappa_s (\theta_{sfc}-\theta_h)/h_s$$
+
+For description and values of the parameters above, please see [parameters_initialconditions.jl](parameters_initialconditions.jl). 
+
+### 2. Implementation
+- for the slab land, we use an explicit time stepping:
+$$T_{sfc}^{n+1} = T_{sfc}^n +\frac{(F_{tot}^{n} + G^{n})}{\rho_s c_s h_s}\Delta t$$
+- this is implemented
+
+        @inline function source!(
+                model::SlabLandModelSetup, source::Vars, state::Vars, gradflux::Vars, aux::Vars, _...)
+            p = model.parameters
+            G    = p.κ_s * (state.T_sfc - p.T_h) / p.h_s # simple soil physics
+            source.T_sfc = - (aux.F_ρθ_prescribed + G) / (p.ρ_s * p.c_s * p.h_s)
+        end
+
+## **Coupler**
+
+### 1. Formulation
+- we're using our sequential coupler, which is described here
+### 2. Surface Flux Handling
+1. Flux calculation
+    - function `calculate_land_sfc_fluxes(_...)` is defined in `coupler/surface_fluxes.jl`, and is used for:
+        - the Atmos Robin boundary condition calculation (see above)
+        - for surface flux accumulation (next point). 
+    - it is called by Atmos, which is assumed to be the model with the shortest timestep. This means that the fluxes are only calculated once and are consistent with Atmos's time stepping.
+
+2. Flux accumulation
+    - In order to integrate fluxes in time consistently with the Atmos time stepper, we define a new prognostic variable `F_ρθ_accum` whose source function accumulates the fluxes, so that $F\_ρθ\_accum = \int F_{tot} dt$
```

Comment:
maybe this language was causing confusion? the source function doesnt accumulate the fluxes, it is the RHS term of the ODE d(F_rho theta_accum)/dt = F_tot

Reply:
Umm, it basically calculates `d (F_ρθ_accum) +=  \int F_{tot} dt`. I will clarify the doc, but yes, accumulating via the source function is not ideal. In the CM that's the only easy option we have, but we will be giving feedback to the CMCore team on this to make it less confusing in the new version :).

Reply:
hey! not being critical, just more clarifying for my own sake. I still dont understand the confusion re: using the language source - it seems as though this is exactly the same usage (a RHS term for a differential equation) as we always use it, except in this case the ODE is d(F_rho theta_accum)/dt = F_tot.

I can chat with Valeria more though since she was the one who brought it up!